econml/inference.py

```diff
@@ -131,3 +131,80 @@ def coef__interval(self, *, alpha=0.1):
 
     def intercept__interval(self, *, alpha=0.1):
         return self.statsmodels.intercept__interval(alpha)
+
+
+class StatsModelsInferenceDiscrete(Inference):
```

Comment:
Can we create a common base class for StatsModelsInference and StatsModelsInferenceDiscrete?  Looks like almost everything is extremely similar and I'd rather not have so much duplicated code, because if we change it in one place we need to remember to change the parallel copy, too.

Reply:
I would move it in a future PR

Reply:
Ok. #143